docker-compose.yml

```diff
@@ -16,7 +16,7 @@ services:
       SERVICE_SECRET: secret
       SERVICE_TOKEN: token
       SERVICE_SLUG: slug
-      SERVICE_OUTPUT_EMAIL: form-builder-developers@digital.justice.gov.uk
+      SERVICE_OUTPUT_EMAIL: form-builder-developers@digital.justice.gov.uk,form-builder-team@digital.justice.gov.uk
```

Comment:
are we sure we want to email the team about acceptance tests?

Reply:
There is no email service on the acceptance test. We are using the output recorder container (using wire mock) to receive those emails. I learn about the output recorder on this PR. 😅